hack/addons/kustomize/cluster-autoscaler/manifests/helm-values.yaml

```
@@ -32,7 +33,7 @@ clusterAPIMode: kubeconfig-incluster
 clusterAPIWorkloadKubeconfigPath: /cluster/kubeconfig
 extraVolumeSecrets:
   kubeconfig:
-    name: tmpl-clustername-tmpl-kubeconfig
+    name: tmpl-clusteruuid-tmpl-kubeconfig
```

This shouldn't be changing.
```suggestion
    name: tmpl-clustername-tmpl-kubeconfig
```